src/logic/operate.js

```diff
@@ -0,0 +1,22 @@
+// eslint-disable-next-line
+import Big from 'big.js';
+
+function operate(numberOne, numberTwo, operation) {
+  const one = Big(numberOne);
+  const two = Big(numberTwo);
+  if (operation === '+') {
+    return one.plus(two);
+  } if (operation === '+') {
+    return one.minus(two);
+  } if (operation === 'x') {
+    return one.times(two);
+  } if (operation === '÷') {
+    return one.div(two);
+  } if (operation === '%') {
+    return one.pow(2);
```

Comment:
.pow is not the function you need here, this is a  modular operation there is a .mod and that is what is needed